### PR TITLE
MagicCap 2.0.0

### PR DESCRIPTION
**Pull request by dxf**
After making all changes to the cask:

- [ x] `brew cask audit --download {{cask_file}}` is error-free.
- [ x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ x] The commit message includes the cask’s name and version.
- [ x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ x] Named the cask according to the [token reference].
- [ x] `brew cask install {{cask_file}}` worked successfully.
- [ x] `brew cask uninstall {{cask_file}}` worked successfully.
- [ x] Checked there are no [open pull requests] for the same cask.
- [ x] Checked the cask was not [already refused].
- [ x] Checked the cask is submitted to [the correct repo].
- [ x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/magiccap.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'magiccap' do
+  version '2.0.0'
+  sha256 :no_check
```
Comment (reitermarkus) [inline Casks/magiccap.rb]: This needs to be set.

File: Casks/magiccap.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  # github.com/magiccap was verified as official when first introduced to the cask
+  url 'https://github.com/magiccap/MagicCap/releases/download/v2.0.0/magiccap-mac.dmg'
```
Comment (reitermarkus) [inline Casks/magiccap.rb]: Interpolate the version here.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/magiccap.rb (hunk 2 above)
Comment (dxf) [reply Casks/magiccap.rb]: Sorry?

Comment (dxf) [reply Casks/magiccap.rb]: Do you mean as in calculate the sha256?

Comment (reitermarkus) [reply Casks/magiccap.rb]: Look at some other casks. Calculate the checksum and use `#{version}` in the URL.

File: Casks/magiccap.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'magiccap' do
+  version '2.0.0'
+  sha256 :no_check
+
+  # github.com/magiccap was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/magiccap.rb]:
```suggestion
  # github.com/magiccap/MagicCap was verified as official when first introduced to the cask
```

File: Casks/magiccap.rb (hunk 1 above)
Comment (bxio) [reply Casks/magiccap.rb]:
```suggestion
  sha256 '8cebb0061062e03e54a71ff120a6c658a41bbf8952aadfd2945fd76808dc2cb6'
```

brew cask audit output:
`
➜ brew cask audit --download magiccap
==> Downloading https://github.com/magiccap/MagicCap/releases/download/v2.0.0/magiccap-mac.dmg
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/142702658/f8bc2480-
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'magiccap'.
audit for magiccap: passed
`

File: Casks/magiccap.rb (hunk 2 above)
Comment (bxio) [inline Casks/magiccap.rb]:
```suggestion
  url "https://github.com/magiccap/MagicCap/releases/download/v#{version}/magiccap-mac.dmg"
```

**Comment by dxf** [comment]
Hey, committed all of you guys' changes

**Pull request merged**